### DISCUSSION TITLE
Get nvim-tree to start automatically when opening a directory with "nvim ."

### DISCUSSION THREAD

**Discussion by sabino-ramirez** [NvChad/NvChad#1465] [Q&A] [answered]
Is there a way to get nvchad to open in the current directory with a [No Name] file so that nvim-tree may start automatically upon opening directory?

I cannot figure this out. With other configs "nvim ." will open nvim in the current directory and open nvim-tree. Here "nvim ." literally opens a file called "."

Thanks in advance!

**Comment by siduck** [comment, marked as answer]
just remove nvim-tree's lazyloading by loading it after `VimEnter` event  and enable open_on_startup option in nvimtree config, not sure about the exact option name but please look into nvimtree config
